### PR TITLE
add recipe to build squizz

### PR DESCRIPTION
**Pull request by bneron**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bneron** [comment]
Hi, I just submit PR for a new recipe which passed  travis @bgruening

File: recipes/squizz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - libgcc # [linux]
+
+test:
+  commands:
+    - squizz -h
+
+about:
+  home: http://ftp.pasteur.fr/pub/gensoft/projects/squizz/
+  summary: Squizz is a sequence/alignment format checker, but it has some conversion capabilities too.
+  license: GNU GPL 2
```
Comment (bgruening) [inline recipes/squizz/meta.yaml]: Can you change this to `GPL 2`

File: recipes/squizz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  name: squizz
+  version: 0.99d
+
+source:
+  fn: squizz-0.99d.tar.gz
+  url: http://ftp.pasteur.fr/pub/gensoft/projects/squizz/squizz-0.99d.tar.gz
+  md5: 12d9b316a7772911ec3baa2e3f6dda12
+
+build:
+  skip: False
```
Comment (bgruening) [inline recipes/squizz/meta.yaml]: This skip can be removed.

File: recipes/watchdog-wms/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+requirements:
+  build:
+      - openjdk >=8
+      - coreutils >=8
+  run:
+      - openjdk >=8
+      - coreutils >=8
+
+test:
+  files:
```
Comment (johanneskoester) [inline recipes/watchdog-wms/meta.yaml]: Please add a test case. This can be trivial, e.g. calling --help.

Comment (johanneskoester) [reply recipes/watchdog-wms/meta.yaml]: @bgruening are test files supported by mulled? Previously, they weren't if I remember correctly.

Comment (bgruening) [reply recipes/watchdog-wms/meta.yaml]: Yes, currently not supported, sorry.

Comment (johanneskoester) [reply recipes/watchdog-wms/meta.yaml]: Thanks @bgruening. @bneron, looks like these files haven't been added by you, right? They are in fact already in the master branch. Seems like github has a hiccup here. I think this is ready to merge.

**Pull request merged**